### ISSUE TITLE
bug(windows): type `aa` gives output `aâ` instead of `â` in the Edge omnibar (formerly also in Chrome omnibar)

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/keyman#5237]
**Describe the bug**

Per Dega keyboard rule, <kbd> a </kbd> + <kbd> a </kbd> = â, but when type aa in the Chrome omnibar on Windows, it instead outputs aâ.

Note that this does not happen when type in Chrome omnibar on macOS Big Sur.

**To Reproduce**

1. Go to 'Chrome'
2. Click on 'Omnibar'
3. Type 'aa'

**Expected behavior**

All outputs should be consistent with the rules defined.

---

**Keyman for Windows:**
 - OS: Windows 10 Pro 21H1 x64
 - Keyman version: 14.0.274
 - Target Application: Chrome Version 91.0.4472.77 (Official Build) (64-bit)

---

**Keyboard**
 - Keyboard name: dega keyboard
 - Keyboard version: 1.0
 - Language name: Rade

**Additional context**

permalink: https://keyman.com/keyboards/dega#expand-languages

**Comment by ermshiperete** [comment]
I can't repro this with Chrome 105.0.5195.102 on Windows 10 (physical machine). Keyman 14.0.286 as well as 15.0.269.

@MakaraSok Can you retest this, please?

**Comment by rc-swag** [comment]
> I can't repro this with Chrome 105.0.5195.102 on Windows 10 (physical machine). Keyman 14.0.286 as well as 15.0.269.
> 
> @MakaraSok Can you retest this, please?
Ok, so this also works for me on Chrome 105.0.5195.127 on Windows 10 and Keyman 16.0.47alpha (so not a release build)
It works in Firefox.
However, for Microsoft Edge version Version 105.0.1343.33 (Official build) (64-bit)  it still has the reported problem but only for the first character pressed.   I checked another keyboard with a rule and got the same result. It seems ominbar is causing focus to be lost after the first keypressed and the context is reset. 
In Summary: 
1 . It is out of our control, 
2. it now only occurs in Microsoft Edge but seeing as it is based on chromium it may soon bring in the change to fix.  
3. It is only the first key pressed but works for every key after that so annoying but not too hard to deal with.

**Comment by ermshiperete** [comment]
The problem are the search suggestions. After typing the `a` Edge suggests and selects an entire word, in my case `amazon.de`. Since Edge/Chrome is a non-compliant app Keyman has to send a backspace to get rid of the `a`. However, since there is the suggested completion, the backspace merely deletes that instead of the `a`. I think there's nothing Keyman can do. The same behavior can be observed anytime Edge makes a suggestion, e.g. with the suggestion `amazon.de` displayed, delete all characters from the end including the `o`, then type <kbd>o</kbd> twice - this will then show `amazoô`.

As a workaround the user can go to settings, select "Privacy, search and services", "Search and connected experiences", "Address bar and search", "Search suggestions and filter", the disable "Show me suggestions from history, favorites and other data on this device using my typed characters".

**Comment by mcdurdin** [comment]
I don't think we should close this. It's an ongoing compatibility bug that impacts end users. Chrome does have a fully-TSF patch in the works (although it seems like it may have stalled, haven't seen any activity for a long time).

It is also possible that #10051 could mitigate this!

However, this is a duplicate of #1051, so will track there.
